### DISCUSSION TITLE
Failed to initialize

### DISCUSSION THREAD

**Discussion by HB8204** [Rem0o/FanControl.Releases#2324] [Q&A]
I dont know anything about coding. I just want to control my fans. It used to work but now every time I restart my pc it fails to initialize and eventually after I close and open fan control 6 or 7 times it fixes itsself. Started doing this on the latest update. If I uninstall and reinstall that fixes it but its annoying. Is this only for people who know how to code? 
<img width="306" alt="image" src="https://github.com/Rem0o/FanControl.Releases/assets/160800523/301a224d-bcb4-45ed-a414-e16476dab4f2">
[Log.txt](https://github.com/Rem0o/FanControl.Releases/files/14366600/Log.txt)

**Comment by Rem0o** [comment]
When you saved that configuration, there was 77 sensors detected. It now only finds 56, so it tries to refresh to get those 77 again.  If your config is just fine without those new missing sensors, simply hit save and it won't do the retry mechanism.

**Reply by HB8204** [reply]
But its not bringing up any fans at all when it does this. 
<img width="785" alt="image" src="https://github.com/Rem0o/FanControl.Releases/assets/160800523/408fb58c-7b2a-489d-b40a-6e2ad91ba097">

**Reply by HB8204** [reply]
<img width="235" alt="image" src="https://github.com/Rem0o/FanControl.Releases/assets/160800523/2e876ad6-1ac5-40d4-b400-6d30966b58a7">

**Comment by HB8204** [comment]
I hit refresh 4 times and the 4th time they showed up 
<img width="953" alt="image" src="https://github.com/Rem0o/FanControl.Releases/assets/160800523/2ff29744-c6ec-4f04-b451-2585543695cf">

Makes no sense.

**Reply by Rem0o** [reply]
Are you using any plugin?
